### ISSUE TITLE
Feature Request: Search through patches

### ISSUE THREAD

**Issue by jgarte** [MitMaro/git-interactive-rebase-tool#626]
Hi,

Is there a way to search?

`/` doesn't work for example

**Comment by MitMaro** [comment]
Good idea! There's already #237, but I think being able to search a patch also makes sense.

I've done some initial work on #237 over the past year, and it should be possible to extend the search functionality to include this as well:

[![IMAGE ALT TEXT](http://img.youtube.com/vi/FXMOk31tfXg/0.jpg)](http://www.youtube.com/watch?v=FXMOk31tfXg)

**Comment by MitMaro** [comment]
I've added searching to the list view in #797 , but I would like to add search to the patch/diff view as well.

Right now, the "linear" search I used in the list view would be too slow for diffs, at least having the search in the main thread. I've spent the past year adding support for threading to the project, and hope to eventually add search that runs in a background thread.
